idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/inspection/InspectionUtilitySyntax.kt

```diff
@@ -12,8 +13,8 @@ interface InspectionUtilitySyntax {
   }
 
   val ArrowPath: Array<String>
-    get() = arrayOf("Kotlin", "Λrrow")
+    get() = arrayOf(MetaIdeBundle.message("proofs.kotlin"), MetaIdeBundle.message("proofs.arrow"))
```

Comment:
I'd say we don't want to localize feature names since people always refers to them by their English name, regardless of the language. You'd never say "Pruebas de tipos". Still we'd likely want to have all strings under the same strings file, so I think using the localization system here is preferred 👍 , but I'd suggest removing the Spanish translations for paths (in case the system is able to fallback to the english one, like Android would do for example). These paths just lead to the library features so they'll likely be always English.